cypress/integration/2_rules.spec.js

```diff
@@ -93,6 +114,20 @@ describe('Rules', () => {
         SAMPLE_RULE.status
       );
 
+      // Switch to YAML editor
+      cy.get(
+        '[data-test-subj="change-editor-type"] label:nth-child(2)',
+        TWENTY_SECONDS_TIMEOUT
+      ).click({
+        force: true,
+      });
+
+      YAML_RULE_LINES.forEach((line) =>
```

Comment:
**Not blocking**, but could you make an `enhancement` issue to improve this portion of the test, and the check on [line 212](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/201/files#diff-38d7c2fdd6835ad84ae7e1986f9e53e4269f0422c55ab8493c7ef842e89dde72R212)? I learned when implementing the `alerts.spec` test that the contents of the `EuiCodeEditor` is segmented into individual lines. [Here](https://github.com/opensearch-project/security-analytics-dashboards-plugin/blob/main/cypress/integration/3_alerts.spec.js#L278), I implemented a test which would iterate through each line to check whether it contained the expected string. Refactoring these tests to use a similar method would give us more confidence that the code editor contains the expected values in the expected format.

Reply:
Cool, thanks for sharing. Just create new issue on it, and will assign it to me in our task tracking
https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues/208

Reply:
Thank you!